trl/trainer/dpo_config.py

```diff
@@ -141,6 +141,8 @@ class DPOConfig(TrainingArguments):
             for saving memory and speeding up training by not computing the logits for all tokens, especially in scenarios
             when working with very long prompts where labels are -ignored (-100).
             [Read more](https://huggingface.co/docs/transformers/main/model_doc/llama#transformers.LlamaForCausalLM)
+        precompute_ref_batch_size (`int`, *optional*, defaults to `None`):
```

Comment:
Can you:

- move this just after `precompute_ref_log_probs`
- mention that if left empty (`None`), `per_device_train_batch_size` is used